### DISCUSSION TITLE
A simple filter/graph to map my connection speed to the internet - any help be great

### DISCUSSION THREAD

**Discussion by sjmgeezer** [Security-Onion-Solutions/securityonion#1854] [Unsupported Versions] [answered]
hopefully an easy one - just wanted to map how my connection to the internet is over time.  See when things are slow and otherwise.
nothing fancy - a simple graph would be awesome.

Yes I'm new - that is learning - I thought this one would be easy - maybe I'm over thinking it.

Any help be great.

Thanks

SM

**Comment by dougburks** [comment, marked as answer]
Assuming your Security Onion installation is monitoring your Internet connection via tap or span port, one option would be to look at the `Monitor Traffic` graph in Grafana to see how much actual traffic is going across that interface:
![image](https://user-images.githubusercontent.com/1659467/98535852-cd2fef00-2254-11eb-8aaa-b8644a199902.png)

For more information about Grafana, please see:
https://docs.securityonion.net/en/2.3/grafana.html

**Comment by sjmgeezer** [comment]
Thanks Doug -great product - awesome support both Community and yourself

Answered
